src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -75,6 +75,7 @@ private Loan(JsonObject representation, Item item, User user, User proxy,
     // and possibly convert isFound to unknown item class
     if (item != null && item.isFound()) {
       representation.put("itemId", item.getItemId());
+      representation.put("itemLocationId", item.getLocationId());
```

Comment:
This will change the JSON representation of the loan every time it is fetched. My understanding is that isn't the intended behaviour for this change.